src/dashboard/src/components/archival_storage/atom.py

```diff
@@ -87,11 +87,11 @@ def add_prop_from_xml(dict_, name, el, xpath):
                 'aip_uuid': aip_uuid,
             }
             amdsec = item.amdsecs[0].serialize()
-            add_prop_from_xml(attrs, 'size', amdsec, './/{info:lc/xmlns/premis-v2}objectCharacteristics/{info:lc/xmlns/premis-v2}size')
-            add_prop_from_xml(attrs, 'format_name', amdsec, './/{info:lc/xmlns/premis-v2}objectCharacteristics/{info:lc/xmlns/premis-v2}format/{info:lc/xmlns/premis-v2}formatDesignation/{info:lc/xmlns/premis-v2}formatName')
-            add_prop_from_xml(attrs, 'format_version', amdsec, './/{info:lc/xmlns/premis-v2}objectCharacteristics/{info:lc/xmlns/premis-v2}format/{info:lc/xmlns/premis-v2}formatDesignation/{info:lc/xmlns/premis-v2}formatVersion')
-            add_prop_from_xml(attrs, 'format_registry_name', amdsec, './/{info:lc/xmlns/premis-v2}objectCharacteristics/{info:lc/xmlns/premis-v2}format/{info:lc/xmlns/premis-v2}formatRegistry/{info:lc/xmlns/premis-v2}formatRegistryName')
-            add_prop_from_xml(attrs, 'format_registry_key', amdsec, './/{info:lc/xmlns/premis-v2}objectCharacteristics/{info:lc/xmlns/premis-v2}format/{info:lc/xmlns/premis-v2}formatRegistry/{info:lc/xmlns/premis-v2}formatRegistryKey')
+            add_prop_from_xml(attrs, 'size', amdsec, './/{http://www.loc.gov/premis/v3}objectCharacteristics/{http://www.loc.gov/premis/v3}size')
+            add_prop_from_xml(attrs, 'format_name', amdsec, './/{http://www.loc.gov/premis/v3}objectCharacteristics/{http://www.loc.gov/premis/v3}format/{http://www.loc.gov/premis/v3}formatDesignation/{http://www.loc.gov/premis/v3}formatName')
+            add_prop_from_xml(attrs, 'format_version', amdsec, './/{http://www.loc.gov/premis/v3}objectCharacteristics/{http://www.loc.gov/premis/v3}format/{http://www.loc.gov/premis/v3}formatDesignation/{http://www.loc.gov/premis/v3}formatVersion')
+            add_prop_from_xml(attrs, 'format_registry_name', amdsec, './/{http://www.loc.gov/premis/v3}objectCharacteristics/{http://www.loc.gov/premis/v3}format/{http://www.loc.gov/premis/v3}formatRegistry/{http://www.loc.gov/premis/v3}formatRegistryName')
+            add_prop_from_xml(attrs, 'format_registry_key', amdsec, './/{http://www.loc.gov/premis/v3}objectCharacteristics/{http://www.loc.gov/premis/v3}format/{http://www.loc.gov/premis/v3}formatRegistry/{http://www.loc.gov/premis/v3}formatRegistryKey')
```

Comment:
This is a good place to use `metsrw.fsentry.get_premis_objects` because you don't have to tell it the namespace you want to use. That would let us read from both v2 and v3. It returns a `PREMISObject` that makes very easy to access the properties, e.g. `premis_object.size`, `premis_object.format_name`, etc...